windows/scripts/MatlabPostBuild.cmd

```diff
@@ -0,0 +1,12 @@
+set SOLUTION_DIR=%~1%
+set OUTPUT_DIR=%~2%
+
+echo MatlabPostBuild.cmd : copy matlab generated scripts to output.
+
+copy /y "%SOLUTION_DIR%..\matlab\+caffe\*.m" "%OUTPUT_DIR%matcaffe\+caffe"
```

Comment:
@mnidza I just saw now your last response. OK, I'll update the scripts as you say.